ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,84 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  operator-registry:
+    name: "4.7"
+    namespace: ocp
+    tag: operator-registry
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: build/Dockerfile.ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/learn-operator
+      - destination_dir: .
+        source_path: /tmp/bin
+  to: learn-operator
+- dockerfile_path: deploy/Dockerfile.src.ci
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: learn-oc-bin-image
```

Comment:
It is no longer necessary to build your own image with `oc`, please use the `cli` stanza in your test. https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

Reply:
Thanks! Removed it.

Reply:
I guess I have to add it back since the `cli` image doesn't contain the `make` command. The log when using the `cli` image:
```console
2020/10/30 08:15:06 Pod learn-oo-test-aws-optional-operators-ci-subscribe succeeded after 1m39s
2020/10/30 08:15:07 Executing pod "learn-oo-test-aws-e2e-test"
2020/10/30 08:15:18 Container cp-secret-wrapper in pod learn-oo-test-aws-e2e-test completed successfully
/bin/bash: line 4: make: command not found
error: failed to execute wrapped command: exit status 127
```

Reply:
No, you would need to have

```
from: src
cli: latest
```

in the step, so the test would run using your `src` image with `oc` injected automatically.

Reply:
Do you mean change the
```console
    test:
    - as: e2e-test
      commands: |
        make deploy_cr
      from: learn-cli
```
to 
```console
    test:
    - as: e2e-test
      commands: |
        make deploy_cr
      from: src
      cli: latest
```
?
Do we have any docs about these flags(`cli: latest`)? I really confused about them.

Reply:
I sent you that exact link above https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli 
Have you read it?

Reply:
Yeah, but I'm not very sure at that time. Thanks! I am completely clear now. Updated it.